internal/scanner/scanner.go

```diff
@@ -41,6 +48,30 @@ func (s *Scanner) ScanNamespace(namespace string) error {
 		fmt.Println(policy.GetName())
 	}
 
+	auditableResources, err := s.resourcesFetcher.GetResourcesForPolicies(context.Background(), policies, namespace)
+
+	if err != nil {
+		return err
+	}
+
+	// TODO this is for debugging, it should be remove in future steps!
+	for _, resource := range auditableResources {
+		fmt.Println("Policies: ")
+		for _, policy := range resource.Policies {
+			fmt.Println(policy.GetName())
+		}
+		for _, resource := range resource.Resources {
+			fmt.Println(resource)
```

Comment:
This is quite verbose when debugging tbf, but it is what it is. Maybe an additional println to separate resources would be nice.